.github/workflows/configurable-benchmark-test.yaml

```diff
@@ -53,8 +53,8 @@ jobs:
           .github/workflows/scripts/${{ github.event.inputs.service_mesh }}_deploy.sh
         shell: bash
 
-      - name: Initiate tmate session
-        uses: mxschmitt/action-tmate@v3
+      #- name: Initiate tmate session

```

Comment:
If it's no longer necessary, please delete the comment.

Reply:
Ok... I'll delete the action step